modules/efk-logging-fluentd-log-rotation.adoc

```diff
@@ -19,23 +19,34 @@ fluentd.log.3  1Mb
 fluentd.log.4  1Mb
 ----
 
+The following example shows the same conditions for Rsyslog.
+
+----
+rsyslog.log     0b
+rsyslog.log.1  1Mb
+rsyslog.log.2  1Mb
+rsyslog.log.3  1Mb
+rsyslog.log.4  1Mb
+----
+
 .Prerequisites
 
 Set cluster logging to the unmanaged state.
 
 .Procedure
 
-You can control the size of the Fluentd log files and how many of the renamed files that {product-title} retains using
+You can control the size of the log collector files and how many of the renamed files that {product-title} retains using
 environment variables.
 
-.Parameters for configuring Fluentd log rotation
+.Parameters for configuring log rotation
 [cols="3,7",options="header"]
 |===
 |Parameter
 |Description
 
-| `LOGGING_FILE_SIZE` | The maximum size of a single Fluentd log file in Bytes. If the size of the *fluentd.log* file exceeds this value, {product-title} renames the *fluentd.log.** files and creates a new *fluentd.log*. The default is 1024000 (1MB).
-| `LOGGING_FILE_AGE` | The number of logs that Fluentd retains before deleting. The default value is `10`.
+| `LOGGING_FILE_SIZE` | The maximum size of of the fluentd.log file or the rsyslog.log file in Bytes. If the size of the *fluentd.log* file exceeds this value, {product-title} renames the log files and creates a new file. The default is 1024000 (1MB).
```

Comment:
The log rotation of rsyslog is different from fluentd. There has a configmap logrotate-crontab controlls the rsyslog's log rotation. You can get some information from: https://bugzilla.redhat.com/show_bug.cgi?id=1730211#c1

Reply:
Yes, there is a configmap with a crontab, and one with the logrotate config, but the user doesn't need to use those.  Most users will be able to set these environment variables and not need to know the implementation details.  I suppose we should document those configmaps, but we should tell users that they probably don't need to touch them.

Reply:
@QiaolingTang @nhosoi In order to change the log rotation parameters, does the user still edit the `rsyslog` daemonset or edit the `logrotate` configmap?
```
oc set env daemonset/rsyslog LOGGING_FILE_AGE=30 LOGGING_FILE_SIZE=1024000

oc set env cm/logrotate LOGGING_FILE_AGE=30 LOGGING_FILE_SIZE=1024000
```

Reply:
This one:
```
oc set env daemonset/rsyslog LOGGING_FILE_AGE=30 LOGGING_FILE_SIZE=1024000
```
If an advanced user needs to change the cron or logrotate settings, they would do
```
oc edit configmap logrotate-bin
or
oc edit configmap logrotate-crontab
```

Reply:
@richm Thanks. I didn't see your comment above as I didn't refresh the page.